plugins/scaffolder/src/components/fields/RepoUrlPicker/validation.ts

```diff
@@ -13,11 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { ApiHolder } from '@backstage/core-plugin-api';
 import { FieldValidation } from '@rjsf/core';
 
 export const repoPickerValidation = (
```

Comment:
Just for extra here, can we do `repoPickerValidation: CustomFieldValidator<string> =` and then we can remove the `_context` argument?

Reply:
Hmmm that made the tests still force the presence of the context due to typescript magic; i just left out the type from this file and cleaned up the tests to not pass in a context instead